### PR TITLE
atomicparsley: add test

### PR DESCRIPTION
**Pull request by redgene**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by redgene** [comment]
@fxcoudert

**Comment by redgene** [comment]
@fxcoudert Please review :)

File: Formula/atomicparsley.rb [hunk 1]
```diff
@@ -35,4 +35,12 @@ def install
                           "--disable-universal"
     system "make", "install"
   end
+
+  test do
+    cp test_fixtures("test.m4a"), testpath/"file.m4a"
+    system "#{bin}/atomicparsley", testpath/"file.m4a", "--artist", "Homebrew"
+    output = shell_output("find . -name file-temp*")
+    var = "file-temp"
```
Comment (fxcoudert) [inline Formula/atomicparsley.rb]: Can we find a test that doesn't rely on internal details, such as looking for temporary files?

**Comment by redgene** [comment]
The main output file is named that way.
```
$ ls
file.m4a

$ atomicparsley file.m4a --artist Homebrew 

 Started writing to temp file.
 Progress: =========================================> 74% --------------|
 Finished writing to temp file.

$ ls -al
total 16
drwxr-xr-x   4 utkarshtripathi  staff   136 Feb  7 18:33 .
drwx------+ 29 utkarshtripathi  staff   986 Feb  7 18:33 ..
-rw-r--r--   1 utkarshtripathi  staff  3761 Feb  7 18:33 file-temp-69313.m4a
-rw-r--r--   1 utkarshtripathi  staff  1689 Feb  7 18:28 file.m4a
```
Another way of testing can be: checking if our output file contains 'Homebrew' as its artist:

```
$ atomicparsley file-temp-69313.m4a --textdata
Atom "©too" contains: Lavf57.41.100
Atom "©ART" contains: Homebrew
```

But we will still be checking its temporary file.

**Comment by fxcoudert** [comment]
Let's use `--overWrite` to avoid the use of tempfile. Then check that the file has Homebrew in it. That would be a sensible test.

(Seriously, who writes output by default to a file named `tempfile`??)

**Comment by fxcoudert** [comment]
Thanks @redgene for the new test!

**Pull request merged**